### ISSUE TITLE
[Suggestion] Add Tesseract and Solar Generation to the pack

### ISSUE THREAD

**Issue by Irgendwer01** [xJon/The-1.12.2-Pack#620]
**If your feature request is a mod suggestion:**
1. Has someone suggested this mod before: no
2. Why should the mod be added: Tesseract ports the old Tesseract from Thermal Expansion for 1.7.10 to 1.12.2 and Solar Generation adds Thermal Expansion Solar Panels.

**Mod info & download page**
* Link to official page:
https://www.curseforge.com/minecraft/mc-mods/tesseract
https://www.curseforge.com/minecraft/mc-mods/solargeneration

**Comment by d3ad-pix3l** [comment]
Tesseracts are my fav!

**Comment by Hexality** [comment]
yeah tesseract pls, its such a pain in the butt to transfer energy and items over dimensions

**Comment by xJon** [comment]
> yeah tesseract pls, its such a pain in the butt to transfer energy and items over dimensions

You can use the Quantum Entangloporter from Mekanism or Dimensional Transceiver from EnderIO.

**Comment by Hexality** [comment]
yeah I can but they're extremely expensive to make, having a interdimensional power distribution without tesseract is simply inviable
I added tesseract extension to the modpack before but every update it gets removed
